web-client/iron-remote-desktop/README.md

```diff
@@ -41,14 +42,14 @@ Other advanced functionalities (sharing / copy past...) are not implemented yet.
 
 ## Component parameters
 
-You can add some parameters for default initialization on the component `<iron-remote-gui />`.
+You can add some parameters for default initialization on the component `<iron-remote-desktop-rdp />`.
 
 > Note that due to a limitation of the framework all parameters need to be lower-cased.
 
 - `scale`: The scaling behavior of the distant screen. Can be `fit`, `real` or `full`. Default is `real`;
-- `verbose`: Show logs from `iron-remote-gui`. `true` or `false`. Default is `false`.
+- `verbose`: Show logs from `iron-remote-desktop-rdp`. `true` or `false`. Default is `false`.
 - `debugwasm`: Show debug info from web assembly. Can be `"OFF"`, `"ERROR"`, `"WARN"`, `"INFO"`, `"DEBUG"`, `"TRACE"`. Default is `"OFF"`.
-- `flexcentre`: Helper to force `iron-remote-gui` a flex and centering the content automatically. Otherwise, you need to manage manually. Default is `true`.
+- `flexcentre`: Helper to force `iron-remote-desktop-rdp` a flex and centering the content automatically. Otherwise, you need to manage manually. Default is `true`.
```

Comment:
```suggestion
- `flexcentre`: Helper to force `iron-remote-desktop` a flex and centering the content automatically. Otherwise, you need to manage manually. Default is `true`.
```